addons/web_dashboard/static/src/js/dashboard.js

```diff
@@ -0,0 +1,264 @@
+odoo.define('web.dashboard', function (require) {
+"use strict";
+
+var core = require('web.core');
+var Widget = require('web.Widget');
+var Model = require('web.Model');
+var session = require('web.session');
+var PlannerCommon = require('web.planner.common');
+var framework = require('web.framework');
+var webclient = require('web.web_client');
+var PlannerDialog = PlannerCommon.PlannerDialog;
+
+var QWeb = core.qweb;
+var _t = core._t;
+
+var Dashboard = Widget.extend({
+    template: 'DashboardMain',
+    events: {
+        'click .o_install_new_apps': 'new_apps',
+    },
+    start: function(){
+        //Add 'gift' instead of 'share' to enable amazone dashboard
+        return this.load(['apps', 'invitations', 'planner', 'share'])
+    },
+    load: function(dashboards){
+        var self = this;
+        var loading_done = new $.Deferred();
+        session.rpc("/dashboard/info", {}).then(function (data) {
+            var deferred_promises = []
+            _.each(dashboards, function(dashboard) {
+                var def = self[dashboard](data);
+                if (def) {
+                    deferred_promises.push(def);
+                }
+                framework.unblockUI();
+            });
+            $.when.apply($, deferred_promises)
+                .then(function() {
+                    loading_done.resolve();
+                });
+        });
+        return loading_done;
+    },
+    apps: function(data){
+        this.$('.o_web_dashboard_apps').append(QWeb.render("DashboardApps", data['apps']));
```

Comment:
App Dashboard(first one) is just simple informative dashboard and it don't having much thing to do unlike other dashboards.